.babelrc

```
@@ -0,0 +1,3 @@
+{
+  "presets": ["babel-preset-react-app"]
```

This doesn't look right. Shouldn't it be 
```suggestion
  "presets": ["react-app"]
```

Also, we won't be using CRA's preset, please change it to what Gilbert used. We're setting up from scratch, even babel.

---

Ok